opencensus-exporter/src/main/java/com/spotify/ffwd/FfwdMetricsConverter.java

```diff
@@ -0,0 +1,182 @@
+import io.opencensus.common.Functions;
+import io.opencensus.exporter.metrics.util.MetricExporter;
+import io.opencensus.metrics.LabelKey;
+import io.opencensus.metrics.LabelValue;
+import io.opencensus.metrics.export.Distribution;
+import io.opencensus.metrics.export.Metric;
+import io.opencensus.metrics.export.MetricDescriptor.Type;
+import io.opencensus.metrics.export.Point;
+import io.opencensus.metrics.export.Summary;
+import io.opencensus.metrics.export.TimeSeries;
+import java.io.IOException;
+import java.net.SocketException;
+import java.net.UnknownHostException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FfwdMetricsConverter extends MetricExporter {
+  private static final Logger LOG = LoggerFactory.getLogger(FfwdMetricsConverter.class);
+
+  private FastForward client;
+  FfwdMetricsConverter(final String ffwdHost, final int ffwdPort) {
+    try {
+      client = FastForward.setup(ffwdHost, ffwdPort);
+    } catch (UnknownHostException | SocketException e) {
+      throw new IllegalStateException("Failed to start ffwd opencensus client", e);
+    }
+  }
+
+  protected FfwdMetricsConverter(final FastForward client) {
+    this.client = client;
+  }
+
+
+  // Constant functions for TypedValue.
+  private static final Function<Double, List<com.spotify.ffwd.Metric>> typedValueDoubleFunction =
+      new Function<Double, List<com.spotify.ffwd.Metric>>() {
+        @Override
+        public List<com.spotify.ffwd.Metric> apply(Double value) {
+          return List.of(new com.spotify.ffwd.Metric().value(value));
+        }
+      };
+
+  private static final Function<Long, List<com.spotify.ffwd.Metric>> typedValueLongFunction =
+      new Function<Long, List<com.spotify.ffwd.Metric>>() {
+        @Override
+        public List<com.spotify.ffwd.Metric> apply(Long value) {
+          return List.of(new com.spotify.ffwd.Metric().value(value));
+        }
+      };
+
+  private static final Function<Distribution, List<com.spotify.ffwd.Metric>>
+      typedValueDistributionFunction =
+      new Function<Distribution, List<com.spotify.ffwd.Metric>>() {
+        @Override
+        public List<com.spotify.ffwd.Metric> apply(Distribution distribution) {
+          //TODO: Big task of support distributions in Heroic.
+          return List.of();
+        }
+      };
+  private static final Function<Summary, List<com.spotify.ffwd.Metric>> typedValueSummaryFunction =
+      new Function<Summary, List<com.spotify.ffwd.Metric>>() {
+        @Override
+        public List<com.spotify.ffwd.Metric> apply(Summary summary) {
+          // TODO: Make the percentiles configurable.
+          final List<Summary.Snapshot.ValueAtPercentile> snapshot =
+              summary.getSnapshot().getValueAtPercentiles();
+
+          // TODO: Add stat: min/max/average/p75/p99
```

Comment:
What needs to be done besides uncommenting the code below?

Reply:
i removed the commented out code for now - the summary code in OC isn't really used/recommended so I don't think it actually works/is fully implemented.